packages/react-components/react-nav-preview/src/components/NavItem/useNavItem.ts

```diff
@@ -43,6 +43,7 @@ export const useNavItem_unstable = (props: NavItemProps, ref: React.Ref<HTMLAnch
         ref,
         role: 'nav',
         type: 'navigation',
+        tabIndex: 0, // makes items focusable
```

Comment:
This should not be needed for any of the components, I think the arrow navigation might be messing it up. We should debug this rather than add tabIndex 0 since <a> is a tabstop by default. Feel free to ping me offline to check it out

Reply:
Yep yep, makes sense.

Created this issue to investigate further in a seperate PR.

https://github.com/microsoft/fluentui/issues/31190